access-control-app/src/cmr/access_control/services/acl_validation.clj

```diff
@@ -41,6 +41,7 @@
                               "CATALOG_ITEM_ACL"                [c r u d]
                               "INGEST_MANAGEMENT_ACL"           [r u]
                               "DATA_QUALITY_SUMMARY_DEFINITION" [c u d]
+                              "DASHBOARD_DAAC_CURATOR_ROLE"     [c r u d]
```

Comment:
I really like this map.

Reply:
Yea, the guy who made it is really smart.